### PR TITLE
Update revs

### PR DESCRIPTION
**Pull request by koltiradw**

### CODE REVIEW

**Comment by koltiradw** [comment]
I`m waiting mass rebuild on my local machine....

File: projects/ndpi/build.sh [hunk 1]
```diff
@@ -112,8 +112,8 @@ mkdir sydr && cd sydr
 $CC $CFLAGS -I $INCLUDE_DIR -I /nDPI/example /nDPI/load_sydr_ndpi_reader.c /nDPI/fuzz/fuzz_common_code.c /nDPI/example/reader_util.c $LIB_DIR/libndpi.a /libpcap-1.9.1/libpcap.a /json-c/sydr/libjson-c.a -lm -o load_sydr_ndpi_reader
```
Comment (apach301) [inline projects/ndpi/build.sh:112]: `fuzz_ndpi_reader` and `fuzz_process_packet` get updated in repo, while corresponding Sydr wrappers stay unchanged.
Why don't build them the same way like `quic_get_crypto` and `tls_certificate` - without separate `load_sydr_*` file?

**Review by apach301 — CHANGES_REQUESTED** [review]

**Review by apach301 — CHANGES_REQUESTED** [review]

**Comment by apach301** [comment]
Also, why sydr and coverage targets prefixed with `load_`?

**Comment by koltiradw** [comment]
@apach301
Iam need a few days for resolve current issues and for update other projects.

**Comment by koltiradw** [comment]
@apach301
I'll close the PR for now. I'll be back in an unspecified time.

**Pull request closed without merging**